ci/stress_tests/application_cluster_template.yaml

```diff
@@ -69,11 +66,11 @@ head_node:
 # http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.ServiceResource.create_instances
 worker_nodes:
     InstanceType: <<<WORKER_TYPE>>>
-    ImageId: ami-0027dfad6168539c7  # Amazon Deep Learning AMI (Ubuntu), Version 21.2
+    ImageId: ami-0757fc5a639fe7666
 
     # Run workers on spot by default. Comment this out to use on-demand.
-    InstanceMarketOptions:
-        MarketType: spot
+#    InstanceMarketOptions:
+#        MarketType: spot
```

Comment:
Why this change?

Reply:
Spot instances (1) may not launch immediately, (2) may be killed at any time. These are both generally bad if launching a small number of instances.

They do have an advantage in that you can launch more without requiring a limit increase, however here what happens is that AWS just sometimes won't fulfill your spot requests if try to request this many without a quota anyways...